app/models/role.rb

```diff
@@ -37,6 +38,10 @@ def access_level
     end
   end
 
+  def set_defaults
+    self.active = true if self.new_record?
```

Comment:
Thanks.  stole it from some guy's blog on best practices for setting defaults in rails